pkg/manager/cache.go

```diff
@@ -7,11 +7,45 @@ import (
 	"fmt"
 	"time"
 
+	"k8s.io/apimachinery/pkg/labels"
 	ctrlcache "sigs.k8s.io/controller-runtime/pkg/cache"
 	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 	ctrlmgr "sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
+// NewLabelSelectorCacheForObject creates a new cache that watches the specified
+// object type selected by the label selector in all namespaces . The cache is
+// added to the manager and starts alongside the other leader-election runnables.
+func NewLabelSelectorCacheForObject(
```

Comment:
Can this PR please add a test to `./pkg/manager/test` for the new label selector cache, similar to the test there for the namespace cache?

Reply:
nm: it appears we don't test the namespace cache after all. Deferring to the controller-runtime test sounds good.